zerver/openapi/zulip.yaml

```diff
@@ -28000,22 +28006,36 @@ components:
         - allow_empty_topic
         - disable_empty_topic
         - empty_topic_only
+      example: inherit
```

Comment:
Since this schema is being used for endpoint parameters, I added this here as otherwise we were showing `null` as the example. This is ignored in when the schema is used for a response value in our rendered docs, though this might not be ideal in general.